### PR TITLE
feat: Seed class

### PR DESCRIPTION
**Pull request by JuanPablo2655**

### CODE REVIEW

**Comment by JuanPablo2655** [comment]
hey @NinjaNas

File: src/main/java/com/hacknc/utils/Seed.java [hunk 1]
```diff
@@ -15,19 +15,26 @@
     public Seed(Type seed) {
         Seed.seed = seed;
```
Comment (NinjaNas) [inline src/main/java/com/hacknc/utils/Seed.java]: static Type _seed

Comment (NinjaNas) [reply src/main/java/com/hacknc/utils/Seed.java]: _seed = seed;

**Review by NinjaNas — CHANGES_REQUESTED** [review]
static Type _seed; -> private Type _seed;

**Pull request merged**